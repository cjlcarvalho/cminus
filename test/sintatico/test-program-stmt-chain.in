void main(void)
{
    if(1 == 1)
        while(1 == 1)
            if(1 == 1)
                return;
            else
                if(2 == 2)
                    return;
                else
                    while(1);
}
