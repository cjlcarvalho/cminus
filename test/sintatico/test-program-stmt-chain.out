[program 
  [fundecl 
    [void]
    [main]
    [paramlist]
    [block 
      [if 
        [== [1] [1]]
        [while 
          [== [1] [1]]
          [if 
            [== [1] [1]]
            [return]
            [if 
              [== [2] [2]]
              [return]
              [while [1]
                [;]
              ]
            ]
          ]
        ]
      ]
    ]
  ]
]
